File: java/library.java
import book;

public class library {

public int libId;
public String libName;
public int scansPerDay;
public int sendtime;
public int books[];

library(int libId, String libName, int scansPerDay, int sendtime, book[] books){

    this.libId=libId;
    this.libName=libName;
    this.scansPerDay=scansPerDay;
    this.sendtime=sendtime;
    this.books=books;
}

public book getBook(book mybook,int key){
	return this.books[key];	
}

public void queueBooks(){
	return this.books	
} 


}

File: library.php
<?php 

/**
 * 
 */
class library
{
	public $id;
	public $name;
	public $scan_per_day;
	public $sign_up_duration;
	public $books;

	function __construct($id,$name,$scan_per_day,$sign_up_duration)
	{
		$this->$id = $id;
		$this->$name = $name;
		$this->$scan_per_day = $scan_per_day;
		$this->sign_up_duration = $sign_up_duration;
		$this->books = array();
		
	}//constructor()

	function getBook($book,$key)
	{
		return $this->books[$key];
	}//scan();

	function queueBooks($books)
	{
		return $this->books=rsort($books);

	}//queueBooks()

}//Class Library





 ?>